sourcegraph/about
# Demand-Gen: Dev Onboarding Landing

This closes #5320. It adds the gated page Developer Onboarding and abstracts the homepage's feature walkthrough into a component.

Reference: [Figma](https://www.figma.com/file/hTjBMaMgCKlkO6OOOThU6r/Dev-Onboarding-DG-landing-page?node-id=0%3A1) and [meta](https://docs.google.com/spreadsheets/d/1JCCIqbP9V6Q4Vwc4yEBJc1wX3y-BHUx32naWRWswWhw/edit#gid=1572625403)

### Changelog

- Adds `better-dev-onboarding`
- Creates a `FeatureWalkthrough` component
- Creates `featureCopy` to store common copy

### Notes
- I appreciate any feedback on the component/copy implementation! Want to make sure I am following our team's preferred pattern here.
- The Figma calls for text CTAs in the walkthrough. I kept the button styling for consistency with the homepage, because the copy and layout are otherwise identical.
- The Figma specifies a different responsiveness for `BlogResourceItem` on mobile: two columns with a small image. This is consistent across other wireframes, but it is not used on Prod. I went with the Prod pattern here. Are we looking to update the mobile design for `BlogResourceItem` to the one shown in Figma?

### Test

1. Navigate to `/better-dev-onboarding`
2. Please check that the design matches the Figma on desktop and mobile
3. Please check that the homepage remains consistent with `FeatureWalkthrough` implemented
4. Ensure linting passes.
5. Ensure prettier has standardized the proposed changes.
6. Ensure dev and production builds work.
